### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by guidolarosa**
- Updated tagline

- Also removed unnecessary animation from footer.

### CODE REVIEW

File: src/utils/ui-constants.js [hunk 1]
```diff
@@ -290,7 +290,7 @@ export const CommunityBannerContent = {
 }
 
 export const CommunityLinksContent = {
-    preHeader: 'Tagline',
+    preHeader: 'Swapr socials',
```
Comment (adamazad) [inline src/utils/ui-constants.js:293]:
```suggestion
    preHeader: 'Swapr Socials',
```

**Review by adamazad — APPROVED** [review]

File: src/utils/ui-constants.js [hunk 2]
```diff
@@ -43,9 +43,9 @@ import Twitter from './../assets/images/isologo-twitter.svg'
 
 // Blog Images
 
-import Entry1 from './../assets/images/blog-entry-1.png'
-import Entry2 from './../assets/images/blog-entry-2.png'
-import Entry3 from './../assets/images/blog-entry-3.png'
+import Entry1 from './../assets/images/blog/blog-entry-1.png'
+import Entry2 from './../assets/images/blog/blog-entry-2.png'
+import Entry3 from './../assets/images/blog/blog-entry-3.png'
```
Comment (adamazad) [inline src/utils/ui-constants.js:48]: I noticed these images are big for the landing page. Can we compress them on [Squoosh](https://squoosh.app/)?

File: src/utils/ui-constants.js [hunk 3]
```diff
@@ -310,7 +311,7 @@ export const CommunityLinksContent = {
             content: <>Contribute to the <br/>Swapr repositories on Github.</>,
```
Comment (adamazad) [inline src/utils/ui-constants.js:311]:
```suggestion
            content: <>Contribute to the <br/>Swapr repositories on GitHub.</>,
```

**Review by adamazad — COMMENTED** [review]
I couldn't comment the exact line in this review, but in `src/utils/ui-constants.js#243`
```
                    href: 'https://swapr.eth.link/#/liquidity-mining/create',
```

should be an internal link using `NavLink` or `Link` from React Router

                    href: '/liquidity-mining/create',

File: src/utils/ui-constants.js (hunk 2 above)
Comment (guidolarosa) [reply src/utils/ui-constants.js]: @adamazad updated the images, thanks!

File: src/components/LandingPageComponents/Stats.js [hunk 4]
```diff
@@ -79,7 +76,7 @@ const Stats = () => {
         swaprPricePromise.then((data) => {
             return data.json();
         }).then((decodedPrice) => {
-            let swaprPrice = decodedPrice.coins[coinCode].price.toString();
+            let swaprPrice = decodedPrice.coins[coinCode].price.toFixed(2).toString();
```
Comment (0xVenky) [inline src/components/LandingPageComponents/Stats.js:79]: change to toFixed(3)

File: src/components/LandingPageComponents/Stats.js [hunk 5]
```diff
@@ -121,9 +118,9 @@ const Stats = () => {
     }, []);
 
     let statsData = {
-        'TVL': tvl,
+        'TVL': tvl + ' M',
         'SWPR PRICE': swaprPrice,
-        'TOTAL VOLUME': totalVolumeUSD,
+        'TOTAL VOLUME': totalVolumeUSD + ' M',
```
Comment (0xVenky) [inline src/components/LandingPageComponents/Stats.js:123]: add $ symbol before the totalVolumeUSD

Comment (guidolarosa) [reply src/components/LandingPageComponents/Stats.js]: Done

File: src/components/LandingPageComponents/Stats.js (hunk 4 above)
Comment (guidolarosa) [reply src/components/LandingPageComponents/Stats.js]: Done

**Comment by 0xVenky** [comment]
Everything looks great now!
1 last comment: the links are not there anymore, 
image

**Comment by guidolarosa** [comment]
> Merging with the existing change. Everything else apart from the comment above looks good for now.

Hey @0xVenky I'll take care of that and make another PR

**Pull request merged**